templates/src-js.js

```diff
@@ -43,21 +43,22 @@ module.exports = answers => {
 		}, {});
 	}
 	/**
-	 * Initialise ${name} component.
+	 * Initialise o-example component/s.
 	 * @param {(HTMLElement|String)} rootElement - The root element to intialise the component in, or a CSS selector for the root element
 	 * @param {Object} [options={}] - An options object for configuring the component
+	 * @returns {Example|Example[]} The newly constructed Example components
```

Comment:
This `Example` should use the given component name
```suggestion
	 * @returns {Example|Example[]} The newly constructed Example components
```

Reply:
Yes 🤦 I copy pasted from the tutorial which makes an Example component